ngraph_bridge/enable_variable_ops/ngraph_rewrite_pass.cc

```diff
@@ -177,8 +177,16 @@ class NGraphVariableCapturePass : public NGraphRewritePass {
 
     // Do variable capture then, if requested, dump the graphs.
     std::set<string> skip_these_nodes = {};
-    TF_RETURN_IF_ERROR(
-        CaptureVariables(options.graph->get(), skip_these_nodes));
+    // TF_RETURN_IF_ERROR(
+    //        CaptureVariables(options.graph->get(), skip_these_nodes));
+    auto sts = CaptureVariables(options.graph->get(), skip_these_nodes);
+    if (sts != Status::OK()) {
+      NGRAPH_VLOG(4) << "-------------------------------------Ypu were "
```

Comment:
Please remove this VLOG

Reply:
I think you can revert back to the original code here which is commented out.